models/inherit_stock.py

```diff
@@ -1,12 +1,9 @@
 # -*- coding: utf-8 -*-
```

Comment:
The file name should be the same as the model in it.

Reply:
Done with commit [c7c3511](https://github.com/macagua/website_shops_map/commit/c7c351154f75935ac369d3ac3e5f243894311142)